opengrok-indexer/src/main/java/org/opengrok/indexer/history/Annotation.java

```diff
@@ -163,10 +163,14 @@ void addLine(String revision, String author, boolean enabled) {
     }
 
     void addDesc(String revision, String description) {
-        desc.put(revision, Util.encode(description));
+        desc.put(revision, description);
     }
 
     public String getDesc(String revision) {
+        return Util.encode(desc.get(revision));
+    }
+
+    public String getDescRaw(String revision) {
```

Comment:
Any possibility to make this class non-HTML aware and move the encoding to `-web` or `.web`?

Reply:
Sure; thought the `getDesc()` is used more widely.